File: C#/begginer/1013.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string[] input = Console.ReadLine().Split(' ');

        int a = int.Parse(input[0]), b = int.Parse(input[1]), c = int.Parse(input[2]);

        if(c > b) {
            b = c;
        }

        int greatestAB = (a+b+Math.Abs(a-b)) / 2;

        Console.WriteLine($"{greatestAB} eh o maior");

    }

}

File: Java/begginer/1013.java
import java.io.IOException;
import java.util.Scanner;
import java.util.Arrays;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        String[] input = scanner.nextLine().split(" ");
        int[] nums = new int[3];
        
        for(int i = 0; i < 3; i++) {
            nums[i] = Integer.valueOf(input[i]).intValue();
        }
        
        System.out.println(Arrays.stream(nums).max().getAsInt() + " eh o maior");
    }
 
}
